### PR TITLE
🧹 AWS: Deprecate `createdTime` in favor of `createdAt`

### PR DESCRIPTION
**Pull request by tas50**
We use createdAt in 35 places and createTime in 6. Add createdAt to these resources and deprecate the existing createTime.

### CODE REVIEW

File: providers/aws/resources/aws.lr [hunk 1]
```diff
@@ -1977,8 +1983,10 @@ private aws.dynamodb.table @defaults("name region") {
   continuousBackups() dict
   // Tags for the table
   tags() map[string]string
-  // Creation time of the table
+  // Deprecated (use `createdAt` instead)
   createdTime time
+  // Creation time of the table
```
Comment (misterpantz) [inline providers/aws/resources/aws.lr]:
```suggestion
  // Date and time the table was created
```

File: providers/aws/resources/aws.lr [hunk 2]
```diff
@@ -2095,8 +2103,10 @@ private aws.rds.dbcluster @defaults("id region") {
   storageType string
   // Current state of the cluster
   status string
-  // Creation date of the RDS cluster
+  // Deprecated (use `createdAt` instead)
   createdTime time
+  // Creation date of the RDS cluster
```
Comment (misterpantz) [inline providers/aws/resources/aws.lr]:
```suggestion
  // Date and time the RDS cluster was created
```

File: providers/aws/resources/aws.lr [hunk 3]
```diff
@@ -1221,8 +1221,10 @@ private aws.elb.loadbalancer @defaults("name region elbType scheme vpcId dnsName
   attributes() []dict
   // Deprecated: Use `vpc` instead
   vpcId string
-  // Date and time when the load balancer was created
+  // Deprecated (use `createdAt` instead)
   createdTime time
+  // Date and time when the load balancer was created
```
Comment (misterpantz) [inline providers/aws/resources/aws.lr]:
```suggestion
  // Date and time the load balancer was created
```

File: providers/aws/resources/aws.lr [hunk 4]
```diff
@@ -1671,8 +1673,10 @@ private aws.cloudfront.function @defaults("name status") {
   arn string
   // Date and time when the function was last updated
   lastModifiedTime time
-  // Date and time when the function was created
+  // Deprecated (use `createdAt` instead)
   createdTime time
+  // Date and time when the function was created
```
Comment (misterpantz) [inline providers/aws/resources/aws.lr]:
```suggestion
  // Date and time the function was created
```

File: providers/aws/resources/aws.lr [hunk 5]
```diff
@@ -1769,8 +1773,10 @@ private aws.s3.bucket @defaults("name location public") {
   publicAccessBlock() dict
   // Whether the bucket still exists (stale reference)
   exists bool
-  // Date the bucket was created
+  // Deprecated (use `createdAt` instead)
   createdTime time
+  // Date the bucket was created
```
Comment (misterpantz) [inline providers/aws/resources/aws.lr]:
```suggestion
  // Date and time the bucket was created
```

File: providers/aws/resources/aws.lr [hunk 6]
```diff
@@ -2235,8 +2245,10 @@ private aws.rds.dbinstance @defaults("id region engine engineVersion") {
   status string
   // Whether minor version patches are applied automatically
   autoMinorVersionUpgrade bool
-  // Creation date of the RDS instance
+  // Deprecated (use `createdAt` instead)
   createdTime time
+  // Creation date of the RDS instance
```
Comment (misterpantz) [inline providers/aws/resources/aws.lr]:
```suggestion
  // Date and time the RDS instance was created
```

**Comment by misterpantz** [comment]
I just made suggestions that'd make them consistent.

**Review by misterpantz — APPROVED** [review]

**Review by chris-rock — APPROVED** [review]

**Pull request merged**